src/renderer/components/ft-input-tags/ft-input-tags.js

```diff
@@ -0,0 +1,60 @@
+import Vue from 'vue'
+import FtInput from '../ft-input/ft-input.vue'
+import FtTooltip from '../ft-tooltip/ft-tooltip.vue'
+
+export default Vue.extend({
+  name: 'FtInputTags',
+  components: {
+    'ft-input': FtInput,
+    'ft-tooltip': FtTooltip
+  },
+  props: {
+    placeholder: {
+      type: String,
+      required: true
+    },
+    label: {
+      type: String,
+      required: true
+    },
+    showActionButton: {
+      type: Boolean,
+      default: true
+    },
+    tagList: {
+      type: Array,
+      default: () => { return [] }
+    },
+    tooltip: {
+      type: String,
+      default: ''
+    }
+  },
+  methods: {
+    updateTags: function (text, e) {
+      // text entered add tag and update tag list
+      if (!this.tagList.includes(text.trim())) {
+        const newList = this.tagList.slice(0)
+        newList.push(text.trim())
+        this.$emit('change', newList)
+      }
+      // clear input box
+      this.$refs.childinput.handleClearTextClick()
+    },
+    removeTag: function (e) {
+      // Remove tag from list
+      let parent = e.srcElement.parentElement
+      if (parent.nodeName === 'svg') {
+        // for when user clicked the "path" rather than "svg"
+        parent = parent.parentElement
+      }
+      const tagName = parent.firstChild.outerText.trim()
```

Comment:
```suggestion
    removeTag: function (tagName) {
```

Reply:
Hi @absidue thanks for taking a look, Your idea does indeed work and simplifies things. have pushed a commit including it.